main.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+from WonderPosition import *
+from WonderWoerter import *
+from WonderBlumen import *
+from WonderUmgebung import *
+from WonderBewegung import *
+from WonderTexte import *
+from WonderInventar import *
+
+
+class Spiel(object):
+    def __init__(self):
+        self.wortVergleicher = WortVergleicher()
+        self.wondertext = WonderText()
+        self.wonderInventar = WonderInventar(self.wortVergleicher)
+        self.umgebungen = UmgebungsGenerator(self.wortVergleicher)      
+        self.bewegung = Bewegung(self.umgebungen.gebeStartUmgebung(),self.wondertext, self.wortVergleicher)
+        self.umgebungen.gebeStartUmgebung().setzeBewegung(self.bewegung)
+        self.DoktormutterPosition = self.umgebungen.gebeDoktormutterPosition()
+        self.umgebung =  self.umgebungen.gebeStartUmgebung()
+
+    def spiele(self):
+
+        self.wondertext.setzeText("=================WonderDoktorandin========================" + "\n")
```

Comment:
Ich würde für bessere Lesbarkeit mich bei langen Wörtern an den camelCase halten, den du auch sonst benutzt und hier im Konkreten schreiben:
```suggestion
        self.wonderText.setzeText("=================WonderDoktorandin========================" + "\n")
```